### PR TITLE
add r-icarus

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-icarus/LICENSE [hunk 1]
```diff
@@ -0,0 +1 @@
...short notice like this when it starts in an interactive mode:      <program>  Copyright (C) <year>  <name of author>     This program comes with ABSOLUTELY NO WARRANTY; for details type `show w'.     This is free software, and you are welcome to redistribute it     under certain conditions; type `show c' for details.  The hypothetical commands `show w' and `show c' should show the appropriate parts of the General Public License.  Of course, your program's commands might be different; for a GUI interface, you would use an "about box".    You should also get your employer (if you work as a programmer) or school, if any, to sign a "copyright disclaimer" for the program, if necessary. For more information on this, and how to apply and follow the GNU GPL, see <https://www.gnu.org/licenses/>.    The GNU General Public License does not permit incorporating your program into proprietary programs.  If your program is a subroutine library, you may consider it more useful to permit linking proprietary applications with the library.  If this is what you want to do, use the GNU Lesser General Public License instead of this License.  But first, please read <https://www.gnu.org/licenses/why-not-l
```
Comment (dbast) [inline recipes/r-icarus/LICENSE:1]: Please remove the license file... it is included via the`'{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'` entry already.

Comment (FernandezMathieu) [reply recipes/r-icarus/LICENSE]: done

File: recipes/r-icarus/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('icarus')"           # [not win]
+    - "\"%R%\" -e \"library('icarus')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=icarus
+  license: GPL-3.0
```
Comment (cbrueffer) [inline recipes/r-icarus/meta.yaml:39]:
```suggestion
  license: GPL-3.0-only
```

**Review by cbrueffer — CHANGES_REQUESTED** [review]
Looks good apart from the license string.

**Review by cbrueffer — APPROVED** [review]

**Pull request merged**